pkgs/development/tools/godeadcode/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchgit
+}:
+
+buildGoModule rec {
+  pname = "deadcode";
+  version = "0.16.1";
+
+  src = fetchgit {
```

Comment:
Or should we use the GitHub mirror https://github.com/golang/tools ?

Reply:
Is this not part of pkgs/development/tools/gotools/default.nix ?

Reply:
It was added in v0.16.1. Looks like `pkgs/development/tools/gotools/default.nix` is on v0.7.0, so maybe the better approach is to upgrade that to v0.16.1?

Reply:
/cc @danderson @SuperSandro2000

Reply:
we don't need to add a package for every go tool there is because there are just to many